### PR TITLE
Fix issue #90

### PR DESCRIPTION
**Pull request by djjcast**
Here's a fix for issue #90.

### CODE REVIEW

File: plugin/incsearch.vim [hunk 1]
```diff
@@ -71,6 +71,8 @@ noremap <Plug>(_incsearch-#)  #
 noremap <Plug>(_incsearch-g*) g*
 noremap <Plug>(_incsearch-g#) g#
 
+noremap <silent> <Plug>(_incsearch-#-searchforward) :<C-u>let v:searchforward=g:incsearch#consistent_n_direction<CR>
```
Comment (haya14busa) [inline plugin/incsearch.vim]: I'm really sorrrrry for the late reply :sweat_drops: 
Can you fix this line to support `c`(c#) operator and visual mode(v#)?

```vim
nnoremap <silent> <Plug>(_incsearch-#-searchforward) :<C-u>let v:searchforward=g:incsearch#consistent_n_direction<CR>
xnoremap <silent> <Plug>(_incsearch-#-searchforward) :<C-u>let v:searchforward=g:incsearch#consistent_n_direction<CR>gv
inoremap <silent> <Plug>(_incsearch-#-searchforward) <Nop>
```

**Comment by djjcast** [comment]
Okay, the code has been updated to support Visual and Operator-pending mode.

File: plugin/incsearch.vim [hunk 2]
```diff
@@ -55,13 +55,27 @@ noremap <expr> <Plug>(incsearch-nohl0) incsearch#autocmd#auto_nohlsearch(0)
...
+map <Plug>(incsearch-nohl-N)  <Plug>(incsearch-nohl)<Plug>(_incsearch-consistent-n-direction)<Plug>(_incsearch-N)
 map <Plug>(incsearch-nohl-*)  <Plug>(incsearch-nohl)<Plug>(_incsearch-*)
 map <Plug>(incsearch-nohl-#)  <Plug>(incsearch-nohl)<Plug>(_incsearch-#)
 map <Plug>(incsearch-nohl-g*) <Plug>(incsearch-nohl)<Plug>(_incsearch-g*)
 map <Plug>(incsearch-nohl-g#) <Plug>(incsearch-nohl)<Plug>(_incsearch-g#)
 
+function! s:consistent_n_direction() abort
+  if g:incsearch#consistent_n_direction
+    " set v:searchforward to 1
+    call setreg(
```
Comment (haya14busa) [inline plugin/incsearch.vim]: It doesn't save search-offset like `/pattern/e`.

**Comment by haya14busa** [comment]
Nice! Thank you for the PR @djjcast :+1:

**Pull request merged**